apply/golden_file_test.go

```diff
@@ -68,10 +66,10 @@ func TestIntegration(t *testing.T) {
 				fs, _, e := util.TestFs()
 				r.NoError(e)
 
-				// copy tc.configFile into the tmp test dir (so that it doesn't show up as a diff)
+				// copy fogg.yml into the tmp test dir (so that it doesn't show up as a diff)
 				configContents, e := afero.ReadFile(testdataFs, fileName)
 				if os.IsNotExist(e) { //If the error is related to the file being non-existent
-					fileName = tc.configFile
+					fileName = "fogg.yml"
```

Comment:
the if condition repeats `afero.ReadFile(testdataFs, "fogg.yml")` twice and became redundant